src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/CertificateAuthority.cs

```diff
@@ -794,6 +790,36 @@ private enum CertStatus
             Revoked,
         }
 
+        //[OverloadResolutionPriority(-1)]
+        //internal static void BuildPrivatePki(
```

Comment:
I assume we get remove the big hunk of commented out code now?

Reply:
Picky, picky :).  Yep